server/cmwell-data-tools/src/main/resources/reference.conf

```diff
@@ -10,6 +10,7 @@ akka {
   logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
 
   http.host-connection-pool {
+    max-connections = 10
```

Comment:
in the dc process it's overridden in application.conf and this config will be ignored for the STP agent mode.
The correct way to solve it is to have a dedicated config section for the data-tools (as I already wrote in the past).